packages/tests/server/__queryClient.ts

```diff
@@ -1,19 +1,9 @@
-import { Logger, QueryClient, QueryClientConfig } from '@tanstack/react-query';
+import { QueryClient, QueryClientConfig } from '@tanstack/react-query';
 
 type Config = Omit<Partial<QueryClientConfig>, 'logger'>;
 export function createQueryClientConfig(config: Config | undefined) {
-  type LogFn = Logger['error'];
```

Comment:
seems like this is all gone now in v5. marked as deprecated in v4 https://tanstack.com/query/v4/docs/react/guides/custom-logger